extensions/sql-migration/src/dialog/validationResults/validateIrDialog.ts

```diff
@@ -0,0 +1,676 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as constants from '../../constants/strings';
+import { validateIrDatabaseMigrationSettings, validateIrSqlDatabaseMigrationSettings } from '../../api/azure';
+import { MigrationStateModel, MigrationTargetType, ValidateIrState, ValidationResult } from '../../models/stateMachine';
+import { EOL } from 'os';
+import { IconPathHelper } from '../../constants/iconPathHelper';
+
+const DialogName = 'ValidateIrDialog';
+
+enum ValidationResultIndex {
+	message = 0,
+	icon = 1,
+	status = 2,
+	errors = 3,
+	state = 4,
+}
+
+export const ValidationStatusLookup: constants.LookupTable<string | undefined> = {
+	[ValidateIrState.Canceled]: constants.VALIDATION_STATE_CANCELED,
+	[ValidateIrState.Failed]: constants.VALIDATION_STATE_FAILED,
+	[ValidateIrState.Pending]: constants.VALIDATION_STATE_PENDING,
+	[ValidateIrState.Running]: constants.VALIDATION_STATE_RUNNING,
+	[ValidateIrState.Succeeded]: constants.VALIDATION_STATE_SUCCEEDED,
+	default: undefined
+};
+
+export class ValidateIrDialog {
+	private _canceled: boolean = true;
+	private _dialog: azdata.window.Dialog | undefined;
+	private _disposables: vscode.Disposable[] = [];
+	private _isOpen: boolean = false;
+	private _model!: MigrationStateModel;
+	private _resultsTable!: azdata.TableComponent;
```

Comment:
It seems that this table is not sized properly, as the "Validation steps" column is too wide whereas the "status" column isn't wide enough: 


![image](https://user-images.githubusercontent.com/11844501/207171854-486e7d0b-b952-421b-94f2-857e5276e782.png)

Reply:
related bug with error count: https://msdata.visualstudio.com/Database%20Systems/_sprints/taskboard/SQL%20Inline%20Migrations/Database%20Systems/Azure%20Semesters/Zinc%20-%20CY2022%202H/2022-12/2022-12.1?workitem=2126138